scripts.py

```diff
@@ -1260,6 +1278,11 @@ def arg_parser(cls, _db):
             help=u'Titles will be imported into a collection with this name. The collection will be created if it does not already exist.',
             required=True
         )
+        parser.add_argument(
+            '--storage-name',
+            help=u'Storage service to use.',
+            required=True
+        )
```

Comment:
Since this script stores both books and covers, I don't think it makes sense to ask for a single storage integration.

Either we should take separate `--book-storage-name` and `--cover-storage-name` arguments, or we should expect everything to be set up beforehand (so no auto-creating a collection).

Since you have to set up the storage integrations ahead of time, I think the simplest thing is to also require that the collection be set up ahead of time. Basically take out this code and the ability to auto-create a collection. If a collection is specified and it's missing one or both storage integrations, then complain.